terraform/services/github-actions-role/data.tf

```
@@ -23,13 +24,21 @@ locals {
       "web-admin",
     ] : [],
   )
+
+  # TODO Replace with cdap-test and cdap-prod when those environments are set up
+  account_env = contains(["dev", "test"], var.env) ? "bcda-test" : "bcda-prod"
 }
 
 # KMS keys needed for IAM policy
 data "aws_kms_alias" "environment_key" {
   name = "alias/${var.app}-${var.env}"
 }
 
+#TODO Replace with cdap-prod and cdap-test when vpcs are in place
+data "aws_kms_alias" "tmp_cdap_sops_environment_key" {
```

We'll want to keep access to account-level keys around, so this won't be temporary, and we moved the TODO comment up to line 28

```suggestion
data "aws_kms_alias" "account_env" {
```